toolkit/tools/pkg/imagecustomizerlib/typeConversion.go

```diff
@@ -99,33 +99,20 @@ func partitionToImager(partition imagecustomizerapi.Partition, fileSystems []ima
 	return imagerPartition, nil
 }
 
-func partitionFlagsToImager(flags []imagecustomizerapi.PartitionFlag) ([]configuration.PartitionFlag, error) {
-	imagerFlags := []configuration.PartitionFlag(nil)
-	for _, flag := range flags {
-		imagerFlag, err := partitionFlagToImager(flag)
-		if err != nil {
-			return nil, err
-		}
-
-		imagerFlags = append(imagerFlags, imagerFlag)
-	}
-
-	return imagerFlags, nil
-}
-
-func partitionFlagToImager(flag imagecustomizerapi.PartitionFlag) (configuration.PartitionFlag, error) {
-	switch flag {
-	case imagecustomizerapi.PartitionFlagESP:
-		return configuration.PartitionFlagESP, nil
+func partitionFlagsToImager(bootPartitionType imagecustomizerapi.BootPartitionType,
```

Comment:
rename function to `partitionBootTypeToImager`

Reply:
Fixed.